### PR TITLE
Command Flags + Bugs + repr in guild and user

### PR DESCRIPTION
**Pull request by AmashiM**

### CODE REVIEW

File: discord/ext/commands/core.py [hunk 1]
```diff
@@ -34,11 +35,54 @@
 from .context import Context
 
 
+__all__ = (
```
Comment (VincentRPS) [inline discord/ext/commands/core.py]: it would be good if this was a list instead

File: discord/ext/commands/core.py [hunk 2]
```diff
@@ -34,11 +35,54 @@
...
+        self.default = default
+        self.required = False
+
+        for key, value in kwargs.items():
+            setattr(self, key, value)
+
+
+
+
+class CommandParser:
```
Comment (VincentRPS) [inline discord/ext/commands/core.py]: maybe name this FlagParser instead?

File: examples/command_flags.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+bot = Bot(command_prefix='!', intents=intents)
+
+@bot.listen("on_ready")
+async def on_ready():
+    print(f"{bot.user} has logged in!")
+
+
+@bot.command()
+async def ping(ctx):
+    await ctx.send('pong!')
```
Comment (VincentRPS) [inline examples/command_flags.py]: maybe add the latency here?

```suggestion
    await ctx.send(f'pong! {round(bot.latency * 1000)}ms')
```

**Review by VincentRPS — CHANGES_REQUESTED** [review]
also unless i did a shit revert it seems like the `__all__`'s are tuples again? maybe change that.

File: examples/command_flags.py (hunk 3 above)
Comment (AmashiM) [reply examples/command_flags.py]: i honestly just copied the command handler script as the base to start implementing but sure thing

**Review by VincentRPS — APPROVED** [review]

**Pull request merged**